sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/implementation/StorageAccountEncryptionStatusImpl.java

```diff
@@ -19,12 +20,10 @@ protected StorageAccountEncryptionStatusImpl(EncryptionServices encryptionServic
     @Override
     public boolean isEnabled() {
         EncryptionService encryptionService = this.encryptionService();
-        if (encryptionService == null) {
-            return false;
-        } else if (encryptionService.enabled() != null) {
-            return encryptionService.enabled();
+        if (encryptionService == null || encryptionService.enabled() == null) {
+            return true;
```

Comment:
Why returning true if encryptionService or encryptionService.enabled() is null ?

Reply:
We are migrating our services to azure-resource SDK and it is always returning true and causing problems.

Reply:
I assume since early last year, one cannot even disable the service encryption?